dietpi/dietpi-software

```diff
@@ -6427,6 +6435,52 @@ _EOF_
 			Download_Test_Media
 		fi
 
+		if To_Install 212 kavita # Kavita
+		then
+			case $G_HW_ARCH in
+				1) local arch='arm';;
+				2) local arch='arm';;
+				3) local arch='arm64';;
+				*) local arch='x64';;
+			esac
+
+			local fallback_url="https://github.com/Kareadita/Kavita/releases/download/v0.7.8/kavita-linux-$arch.tar.gz"
+			Download_Install "$(curl -sSfL 'https://api.github.com/repos/Kareadita/Kavita/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/kavita-linux-$arch\.tar\.gz\"/{print \$4}")" /opt
+
+			# User
+			Create_User -g dietpi -d /mnt/dietpi_userdata/kavita kavita
+
+			# Data
+			G_EXEC mkdir -p /mnt/dietpi_userdata/{ebooks,comics}
+
+			# Service # https://wiki.kavitareader.com/en/install/linux-install
+			cat << _EOF_ > /etc/systemd/system/kavita.service
+[Unit]
+Description=Kavita Server (DietPi)
+Wants=network-online.target
+After=network-online.target remote-fs.target
+
+[Service]
+SyslogIdentifier=Kavita
+User=kavita
+WorkingDirectory=/opt/Kavita
```

Comment:
```suggestion
WorkingDirectory=/mnt/dietpi_userdata/kavita
```
Not sure whether all configs are written to the executing users home dir, but if the working directory is used at all for any variable files, then those should be `/mnt/dietpi_userdata/kavita`, so that `/opt/kavita` can be safely removed on reinstalls without loosing any configs/data.